libdnf/rpm/solv_query.cpp

```diff
@@ -836,15 +836,15 @@ SolvQuery & SolvQuery::ifilter_sourcerpm(const std::vector<std::string> & patter
                     if (strncmp(c_pattern, name, name_len) != 0) {  // early check -> performance
                         continue;
                     }
-                    auto * sourcerpm = solv::get_sourcerpm(pool, candidate_id);
+                    auto * sourcerpm = solv::get_sourcerpm(pool, {candidate_id});
```

Comment:
Shall we keep PackageId type  for an  argument for functions in solv/package_private.hpp? What about to replace it with `Id`?

Reply:
Done.